x-pack/test/plugin_api_integration/test_suites/task_manager/task_manager_integration.js

```diff
@@ -204,5 +251,6 @@ export default function ({ getService }) {
       expect(Date.parse(currentTask.runAt) - originalRunAt).to.be.greaterThan(expectedDiff - buffer);
       expect(Date.parse(currentTask.runAt) - originalRunAt).to.be.lessThan(expectedDiff + buffer);
     }
+
```

Comment:
no es-lint error on an empty line here! OMG! hahaha

Just a note that although `expectReschedule()` looks a bit horror-show in terms of fragility, it appears given the huge intervals provided in the tests, seems ok to me, and I can't quite think of a better way to do this off the top of my head ...

Reply:
Yeah, I couldn't find a way to make it less fragile, so left it as it was before 😬
Time based FPs are very hard